terraform/gitops/generate-files/templates/mojaloop/kustomization.yaml.tpl

```diff
@@ -16,6 +16,7 @@ helmCharts:
   valuesFile: values-mojaloop.yaml
   namespace: ${mojaloop_namespace}
   additionalValuesFiles:
+  - values-mojaloop-patch.yaml
```

Comment:
Looks like we can move some of the values from values-mojaloop in this patch file. Using the word `patch` does not sound very good to me, sounds like we are fixing some issue with that file.  Maybe use `override`. I was also wondering what is the `addons` file below for.